### DISCUSSION TITLE
LR 11.3.1 + Xtouch Mini no connection to LR

### DISCUSSION THREAD

**Discussion by oni2323** [rsjaffe/MIDI2LR#653] [Problems]
Hello everyone!

I'm having trouble getting MIDI2LR working. 

I'm on an M1 mac, with LR 11.3.1, with MIDI2LR 5.3.0.0 (reverted to check but later version aren't working either). Using an Xtouch mini.

**The plugin is installed and running, but I get a "not connected to lightroom" error.** 

The knobs are recognized but the plugin when they are turned, so it seems to be fine on that side. 

The firewall is turned off on my mac.

Would anyone have any ideas? 

I've attached the log file and a screenshot. 

![Screen Shot 2022-05-04 at 14 40 32](https://user-images.githubusercontent.com/104907344/166683293-a30b407d-8b2e-4497-b723-28e15ab15067.jpg)

[MIDI2LR.log](https://github.com/rsjaffe/MIDI2LR/files/8620389/MIDI2LR.log)

**Comment by rsjaffe** [comment]
Did you check [FAQ 7](https://github.com/rsjaffe/MIDI2LR/wiki#faqknown-issues)? If that doesn't help, post your _MIDI2LRinfo.txt_ contents, which is in the same directory as _MIDI2LR.log_.

**Comment by oni2323** [comment]
thanks for your answer! I did check, and I don't have a firewall on on my mac. I did open the ports through terminal anyways, but it didn't work either.

I don't have a MIDI2LRinfo.txt file in the same folder as MIDI2LRinfo.txt. 

![Screen Shot 2022-05-05 at 11 59 27](https://user-images.githubusercontent.com/104907344/166901611-f631eea3-c918-4b5b-a352-633de6707fb9.jpg)

**Comment by stevenle123** [comment]
I'm getting this error - "Not Connected to Lightroom". Version 6.1.0.0. I disabled and re-enabled LR. Tried restarting the software and computer. Still no luck

"An error occurred while attempting to run one of the plug-in’s scripts.
?:0: attempt to call method 'send' (a nil value)

**Comment by antzpantz** [comment]
Back up your settings, uninstall and reinstall the plugin

**Reply by gabob23** [reply]
Done many times, doesn't work sadly

**Comment by unimatrixZxero** [comment]
Have you tried resetting your x-touch mini? 
1. unplug
2. hold down layer-A and layer-B buttons
3. plug in
4. wait for blinking to finish
5. done

**Reply by gabob23** [reply]
Thank you for the advice, just did that, it reset just fine, but still isn't working. It seems the issue is between lightroom and midi2lr not communicating. Really sucks as it would be an invaluable tool in my work.
